### ISSUE TITLE
Label resizing with dynamically loaded fonts

### ISSUE THREAD

**Issue by tomzx** [knsv/mermaid#255]
Hi,

This is more of a heads up than an actual issue (and this is probably more related to how `dagre` renders labels, but this may be of interest for `mermaid`), but if you use dynamically loaded fonts that are loaded through CSS, such as Google fonts, `mermaid` should wait for the whole page to have been loaded (dom + assets, particularly the font files).

Thus, developers that integrate `mermaid` into their own page should use

```
$(window).load(function() {
	mermaid.initialize();
});
```

over

```
$(document).ready(function() {
	mermaid.initialize();
});
```

Not doing so will most likely result in mermaid rendering graphs that have labels out of bounds.

**Using ready**
![2015-11-11_20-43-54](https://cloud.githubusercontent.com/assets/188960/11108465/fb1450e4-88b4-11e5-9234-5b2a32548b0f.png)

**Using load**
![2015-11-11_20-43-38](https://cloud.githubusercontent.com/assets/188960/11108464/fb0d4c86-88b4-11e5-8e7c-747a3dea9add.png)

**Comment by knsv** [comment]
Thanks @tomzx. That's a good point. I think this should be pointed out in the docs.

Also I think the domContentLoaded used by mermaid should be changed to loadEvent.

So good input, thanks.

**Comment by knsv** [comment]
Using the load event in mermaid now.

**Comment by knsv** [comment]
Closing issue as it's resolution is included in release 0.5.6. Let me know if you do not agree.
